### DISCUSSION TITLE
v1.27.0

### DISCUSSION THREAD

**Discussion by mikecao** [umami-software/umami#1004] [General]
## Features
- New mobile navigation menu
![Animation4](https://user-images.githubusercontent.com/477975/156497186-9db2933d-b9da-4ac6-ac72-2592a7dcc744.gif)

- Added language and theme settings to profile settings
<img src="https://user-images.githubusercontent.com/477975/156503249-8d0bf03d-e3e3-41df-8dae-a65afffad687.png" width="400" />

- New card layout for mobile
<img src="https://user-images.githubusercontent.com/477975/156503769-ecaafb67-8fdd-488f-a90b-566b8aa36f89.png" width="400" />

## Updates
- Dashboard now shows 10 websites at a time
- Don't show metric change indicator when metric delta is 0
- Language translation updates
- Replaced `react-redux` with `zustand`
- Updated Next.js to `12.1.10`

## Fixes
- Fixed issue with `TRACKER_SCRIPT_NAME` not working during runtime
- Fixed setting default date range
- Fix overflowing text in tables
- Fix realtime metrics header overflow for mobile
- Fixed refresh button not updating date range

A huge thanks to all the contributors on this release! 🙏@0xflotus @christopherpickering @sophiabits @JeremDlcn @nfn @Tox46 

<hr /><em>This discussion was created from the release <a href='https://github.com/mikecao/umami/releases/tag/v1.27.0'>v1.27.0</a>.</em>

**Comment by tomajask** [comment]
Hi,

I don't know what has changed, but after upgrading the docker image to the newest version of `postgresql-latest` I'm getting endless loop of 301 redirects... I had to use specific version `postgresql-1e4c3eb` to make it work again.

I use `umami` behind `traefik`. I see some beta middleware warning in the Umami logs:
```
umami_1  | warn  - using beta Middleware (not covered by semver) - https://nextjs.org/docs/messages/beta-middleware
```

Would you mind taking a look at this?

**Comment by mikecao** [comment]
@tomajask are you using the `FORCE_SSL` environment variable? That's the only thing in the middleware that does a redirect.

**Reply by tomajask** [reply]
@mikecao you're right! After removing `FORCE_SSL` env var it started working with v1.27.0. Thanks a lot! 🚀

**Reply by aliifam** [reply]
how to remove FORCE SSL env in railway app?

**Reply by CanRau** [reply]
Can't get my custom domain working on Fly.io, have Cloudflare in front of it, but `FORCE_SSL` seems to cause the redirect loop, tried setting `FORCE_SSL=false` in my fly.toml though no change.
I'm on Umami v1.39.5 and used the [official Docker image](ghcr.io/umami-software/umami:postgresql-latest) on fly.dev domain it works as expected

Just tried `FORCE_SSL = 0` inspired by https://umami.is/docs/environment-variables tho same result

Also just disabled orange proxy cloud on Cloudflare, although in other cases this works fine for me with fly, I still get the same result, whole setup was really straight forward just this custom domain is now holding me back 🥲

**Works now**, might've been the orange cloud and some DNS caching taking longer or DNS cache in general not sure

Okay retried and for me it's **only with grey cloud** (unproxied), might be due to other Cloudflare settings I have, not sure right now.
